src/core/components/ChatView.test.tsx

```diff
@@ -20,6 +20,20 @@ vi.mock(
 );
 
 describe('ChatView', () => {
+  beforeEach(() => {
```

Comment:
isMobile is currently only used in this one test so it's mocked here for now, but this could be moved to somewhere shared if needed in future tests